### PR TITLE
hotfix: update changelog

### PR DESCRIPTION
**Pull request by Simperfit**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets |
| License       | MIT
| Doc PR        |

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 2.0.3
+
+* Fix a bug when handling invalid IRIs
+* Allow to have a property called id even in JSON-LD
+* Exclude static methods from AnnotationPropertyNameCollectionFactory
+* Improve compatibility with Symfony 2.8
```
Comment (soyuka) [inline CHANGELOG.md]: * Add a trait to share composite identifier behavior in data providers
?

Comment (Simperfit) [inline CHANGELOG.md]: This is in master IIRC ?

Comment (soyuka) [inline CHANGELOG.md]: yes

Comment (Simperfit) [inline CHANGELOG.md]: this is for 2.0 only, so it's not in it

**Comment by dunglas** [comment]
Thanks @Simperfit

**Pull request merged**